bundle/edu.gemini.p1monitor/src/main/resources/edu/gemini/p1monitor/config/conf.production-2015A-kr.xml

```diff
@@ -1,5 +1,5 @@
 <conf>
-    <smtp>smtp.cl.gemini.edu</smtp>
+    <smtp>sbfsmtprelay-lv1.cl.gemini.edu</smtp>
 
```

Comment:
Is there a place on the wiki where these changes are documented, and a list of the hosts that are allowed to send mails?